src/main/java/org/osgeo/proj/Proj.java

```diff
@@ -197,6 +197,40 @@ public static CoordinateOperationFactory getOperationFactory(final CoordinateOpe
      * @throws IllegalArgumentException if the specified type is not one of the above-listed types.
      */
     public static <F extends Factory> F getFactory(final Class<F> type) {
+        return getFactory(type, "");
+    }
+
+    /**
+     * Returns a factory of the given type. This method recognizes three groups of factories:
```

Comment:
Should be something like "Returns a factory of the given type for codes allocated by the given authority". I suggest to not copy-and-paste the remaining of the Javadoc, but instead refer to existing javadoc. The intend is to have only one place to edit. For example:

```
 * The valid values for the {@code type} argument are documented in {@link #getFactory(Class)}.
 * The valid values for the {@code authority} argument are documented in {@link #getAuthorityFactory(String)}.
```